MusicNotationKit/MusicNotationKit/Staff.swift

```diff
@@ -42,19 +42,35 @@ public struct Staff {
      
      - parameter measure: The measure to be inserted.
      - parameter index: The index where the measure should be inserted.
+     - parameter inRepeat: Default to false. True when you want the measure to be inserted into the repeat at the
+        given index. False if you want the measure to be inserted either before a repeat or there is no repeat
+        at the specified index.
      - throws:
         - `StaffError.measureIndexOutOfRange`
+        - `StaffError.noRepeatToInsertInto`
+        - `StaffError.hasToInsertIntoRepeatIfIndexIsNotFirstMeasureOfRepeat`
         - `StaffError.internalError`
         - `MeasureRepeatError.indexOutOfRange`
         - `MeasureRepeatError.cannotModifyRepeatedMeasures`
      */
-    public mutating func insertMeasure(_ measure: Measure, at index: Int) throws {
+    public mutating func insertMeasure(_ measure: Measure, at index: Int, inRepeat: Bool = false) throws {
         let notesHolderIndex = try notesHolderIndexFromMeasureIndex(index)
         // Not a repeat, just insert
         if notesHolderIndex.repeatMeasureIndex == nil {
+            guard !inRepeat else {
```

Comment:
After discussing with @migue48, We decided to remove this check and just silently work by ignoring the inRepeat value since it doesn't matter.